storage/src/main/java/tech/pegasys/teku/storage/store/Store.java

```diff
@@ -178,7 +178,7 @@ private Store(
                         .getSignedBeaconBlock()
                         .map((b) -> Map.of(b.getRoot(), b))
                         .orElseGet(Collections::emptyMap)),
-            fromMap(this.blocks),
```

Comment:
not sure if we added fromMap for the blocks access, does it need to be removed?

Reply:
I don't understand you
I'm changing `fromMap` to `fromMapWithLock` in this PR
do you mean that we should completely remove fromMap from provider?

Reply:
was more of a question of if fromMap can be cleanly removed from code altogether now...

looks like its used elsewhere still, so we can't - all good.

Reply:
Ahh yes, it's used by other providers